### DISCUSSION TITLE
Is there any chances to bring Prompt back?

### DISCUSSION THREAD

**Discussion by vasile-encord** [remix-run/react-router#10923] [Proposals]
Is there any plans to bring Prompt back? useBlocker is unstable and not working properly.

**Comment by brophdawg11** [comment]
We have no plans to bring back a `Prompt` component because you can write your own component around `unstable_usePrompt` quite easily.  The `unstable_` prefix is unlikely to ever be removed from `unstable_usePrompt` because it is impossible to get consistent/correct behavior across browsers since they all handle `window.confirm` differently so that API can never be fully guaranteed to work the same way in your app for all of your users.

I do think that we could eventually consider removing the `unstable_` prefix from `unstable_useBlocker`.  Please open Github issues with reproductions if that API is not working for you.
